src/olympia/reviewers/tests/test_views.py

```diff
@@ -8098,14 +8112,15 @@ def setUp(self):
         self.expected_addons = [listed_addon, unlisted_addon]
 
     def test_results(self):
-        response = self.client.get(self.url)
+        with self.assertNumQueries(30):
```

Comment:
30 seems a lot.  Does it at least scale? (i.e. it's always 30 regardless of whether there's 1 or 100 items in the list)  Can you increase the number of addons (or whatever) and call it again to confirm?

Reply:
The patch actually just increases the queries by 1. It's 24 before this patch, 25 with it, and 30 in tests because safepoints.

Reply:
that's fine.  Can you just add a second response.get with more addons to confirm it doesn't increase to 31 (or more)

Reply:
How do I do that without duplicating the test?

Reply:
Add another addon (and add any matching flags) then `with self.assertNumQueries(30):  self.client.get(self.url)` again.  Put it at the end of the test if it'll disrupt the other asserts.

Reply:
Got it, thank you. Fixed in aa9505b3ec.